android/SherpaOnnxJavaDemo/app/src/main/java/com/k2fsa/sherpa/onnx/MainActivity.java

```diff
@@ -0,0 +1,54 @@
+package com.k2fsa.sherpa.onnx;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+import androidx.lifecycle.ViewModelProvider;
+
+import android.Manifest;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.widget.TextView;
+
+import com.k2fsa.sherpa.onnx.service.SpeechSherpaRecognitionService;
+
+import pub.devrel.easypermissions.EasyPermissions;
+
+public class MainActivity extends AppCompatActivity {
+    private AppViewModel appViewModel;
+    private TextView tvText;
+    private static final int RC_AUDIO_PERM = 123;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+        tvText = findViewById(R.id.text);
+        requestMicrophonePermission();
+    }
+
+
+
+    private void startSpeechService(){
+        // 启动语音识别服务
+        Intent serviceIntent = new Intent(this, SpeechSherpaRecognitionService.class);
+        ContextCompat.startForegroundService(this, serviceIntent); // 启动前台服务（如果需要）
+        appViewModel = new ViewModelProvider(Application.getInstance()).get(AppViewModel.class);
+        // 观察语音识别结果
```

Comment:
Could you remove the Chinese comments?

The code is self-explanatory and I think there is no need to add comments here.